src/Microsoft.ML.LightGBM/LightGbmArguments.cs

```diff
@@ -401,13 +407,13 @@ public enum EvalMetricType
             }
             if (!string.IsNullOrEmpty(metric))
                 res["metric"] = metric;
-            res["sigmoid"] = "0.5";
```

Comment:
Have benchmarks been run on this change? I think changing the sigmoid hyperparameter should have an effect. Without testing I don't know that 0.5 is better or worse than 1.0. @guolinke, how much benchmarking was done on your side?

In general, we are well setup to benchmark these changes and we don't know the extent which the source ML algos were benchmarked, so I don't think we should automatically echo the default hyperparameters set upstream. Automated daily regression tests are a stronger answer to changes like this, though I think kicking off a manual test is appropriate in the meantime.

Reply:
I have reverted the default. If someone wants to use sigmoid=1 they can set it now.

Reply:
This *will* have a change, but if it's the default value in the `LightGBM` codebase, then I would suggest we keep it as 0.5. This is adding regularization to the gradient, whereas a value of `1` is basically adding no regularization.

@guolinke Was the value of `0.5` used to cancel out the factor of `2` used in the LambaRank-style loss? This value of `0.5` seems like it was put in to make sure the classifier learned the exact Logit. Does that sound right?